js/viz/series/area_series.js

```diff
@@ -84,9 +89,10 @@ const baseAreaMethods = {
             border: borderStyle,
             elements: {
                 stroke: 'none',
-                fill: options.color || defaultColor,
+                fill: extractColor(options.color) || defaultColor,
                 hatching: options.hatching,
-                opacity: options.opacity
+                opacity: options.opacity,
+                filter: options.lightening || null
```

Comment:
I'll not leave similar notes more, but i recommend checking all places when u use `||` for fallback - it's just not safe
```suggestion
                filter: options.lightening ?? null
```